.env.sample

```diff
@@ -0,0 +1,47 @@
+MAGENTO_VERSION=2
+
+SQ_M2_CORE_VERSION=local
```

Comment:
Leaving **local** in **SQ_M2_CORE_VERSION** did not install any plugin, if you change it to * installs the latest version which I think that it could be the better option. You can still modify your .env file if you need to test a specific version.

Reply:
changed!